addons/binding/org.openhab.binding.modbus/ESH-INF/thing/bridge-tcp.xml

```diff
@@ -23,6 +23,12 @@
 				<description>Slave id. Also known as station address or unit identifier.</description>
 				<default>1</default>
 			</parameter>
+			
+			<parameter name="enableDiscovery" type="boolean">
```

Comment:
Please mention this in thing configuration section in docs

Reply:
done